packages/neuron-wallet/tests/key.test.ts

```diff
@@ -8,17 +8,33 @@ describe('Key tests', () => {
 
   it('import key from mnemonic', async () => {
     const key = Key.fromMnemonic(mnemonic, '1qaz.2wsx', 17, 3)
-    expect(privateKey).toBe(key.getPrivateKey())
-    expect(key.getAddresses()!.receive.length).toEqual(17)
-    expect(key.getAddresses()!.change.length).toEqual(3)
-    expect(key.getAddresses()!.receive[0]).not.toBe(undefined)
+    if (key.keysData === undefined) {
```

Comment:
IMO, throwing errors in tests is unnecessary

Reply:
OK